assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -6,176 +6,163 @@
-                                    + Add Diner</a>
-                            {% endif %}
-                        </td>
-                        <td>
-                            {% if can_edit_stats or can_edit_pay %}
-                                <input type="submit" class="btn btn-block btn-primary" value="Update User Stats"/>
-                            {% endif %}
-                        </td>
-                        {% if show_delete_column %}
-                        <td> <!-- Empty space --></td>
+                {# Footer buttons #}
+                <tr class="d-none d-sm-table-row">
+                    <td>
+                        {% if dining_list|can_add_others:user %}
+                            <a class="btn btn-primary btn-block" href="
+                            {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
+                                + Add Diner</a>
                         {% endif %}
-                    </tr>
+                    </td>
+                    <td>
+                        {% if can_edit_stats or can_edit_pay %}
+                            <button type="submit" class="btn btn-block btn-primary" form="statsForm">
+                                Update User Stats
+                            </button>
+                        {% endif %}
+                    </td>
+                    <td></td>
+                </tr>
 
-                    </tbody>
-                </table>
-                {% csrf_token %}
-            </div>
+                </tbody>
+            </table>
         </div>
-        <div class="row d-sm-none">
-            <div class="col-12">
-                {% if can_edit_stats or can_edit_pay %}
-                    <input type="submit" class="btn btn-block btn-primary" value="Update User Stats"/>
-                {% endif %}
-               {% if dining_list|can_add_others:user %}
-                   <a class="btn btn-primary btn-block" href="
-                    {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
-                       + Add Diner</a>
-               {% endif %}
+    </div>
+    <div class="row d-sm-none">
+        <div class="col-12">
+            {% if can_edit_stats or can_edit_pay %}
+                <button type="submit" class="btn btn-block btn-primary" form="statsForm">
+                    Update User Stats
+                </button>
+            {% endif %}
+           {% if dining_list|can_add_others:user %}
+               <a class="btn btn-primary btn-block" href="
+                {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
+                   + Add Diner</a>
+           {% endif %}
 
-            </div>
         </div>
-    </form>
+    </div>
 
 {% endblock details %}
```

Comment:
I noticed that for some reason, on my desktop, the mouse doesn't hover to the hand/finger anymore when hovering the buttons. Any idea why?

Reply:
I notice it indeed, only change is that it's not directly inside form tags anymore. I'll fix the pointer.

Reply:
Actually, it's not a pointer (hand) currently on the live server as well, since it's a label tag and not a button which doesn't get a pointer on hover. I'll still change it.

Reply:
I'll make an issue instead and we'll fix it in a separate PR